### ISSUE TITLE
Make "Deleted puzzles" section less intrusive

### ISSUE THREAD

**Issue by jpd236** [deathandmayhem/jolly-roger#2278]
We noticed a couple of quirks with the "Deleted puzzles" section that made it more visible than we felt it should be:

- The contents of the section are expanded by default. Since deleted puzzles are meant to stay out of view (otherwise, why delete them?), this seems like an odd choice.
- JR doesn't remember the expanded/collapsed setting, so even if you go and collapse them, they expand again upon reloading the page.

**Comment by zarvox** [comment]
Non-operators don't see deleted puzzles at all.  For a little context: we've historically been willing to tolerate things being a little more potentially-confusing for our operators, who have mostly been expert users and are willing to deal with a little more jank or information overload to avoid missing things.  During Hunt, we generally expect our operators to keep abreast of if there was a duplicate-puzzle-creation mishap, and to be aware of this enough to help explain the situation or extract any data from the deleted spreadsheet, so between wanting the existence of deleted puzzles to be visible to operators by default for awareness and wanting to be consistent with the other groups on the puzzle list page (which are all expanded by default in their initial state), and because the default state doesn't matter until there are actual deleted puzzles in the section to be aware of, I'd still lean towards leaving the deleted puzzles expanded by default.

All that said: it is weird that we don't remember the expanded/collapsed state, and I would totally accept a PR that made sticky expand/collapse behave correctly, which would make it one click away to hide deleted things and keep them hidden.

Of course, this all becomes somewhat moot if we do a good job of avoiding duplicate puzzle entries that need deletion in the first place 😅

**Comment by jpd236** [comment]
I think #2289 should make expand/collapse behave correctly. I'm curious to know if I've missed a reason.

This raises another question. Is it intentional that when you search for puzzles, the deleted section isn't filtered at all by the search? I would expect it to only show deleted puzzles that match the query. But I'm not sure if this is intentional under the reasoning that operators should be aware of everything for some reason.
